src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -310,10 +310,28 @@ def _handle_created(self, event, organization, repo, is_apps):
         user = pull_request['user']
         merge_commit_sha = pull_request['merge_commit_sha']
 
-        # get author TODO(Maxbittker): do we need to use the more complicated
-        # logic from the commit webhook?
-        author = author = CommitAuthor.objects.get_or_create(
+        author_email = u'{}@localhost'.format(user['login'])
+        try:
+            commit_author = CommitAuthor.objects.get(
+                external_id=get_external_id(user['login']),
+                organization_id=organization.id,
+            )
+            author_email = commit_author.email
```

Comment:
nit, but i'd put this line in an `else` block since it's not going to cause a `DoesNotExist`